### DISCUSSION TITLE
Controls settings

### DISCUSSION THREAD

**Discussion by Roman-Kryvenchenko** [Sam-Bowden/drone_game#2] [Ideas]
Hi, I would like to add settings for controls. 

I'm new to rust so it'll be a great practice for me, also it won't be an issue in the case if someone has different keyboard settings :)

**Comment by Sam-Bowden** [comment]
Yeah the game could really do with this, I was going to implement this some time in the future but go right ahead, it will definitely be some good practice. If you need help with Rust or anything else, feel free to email me, I'll be glad to help

What are your thoughts for this settings menu? The game could do with a pause menu as well, but that's something else for the future, for the moment we could just add a settings button to the first main menu, opening a separate page (and then potentially add a button to this page in a pause menu as well later on).

**Reply by Roman-Kryvenchenko** [reply]
While I was playing it was frustrating to click button to restart the game. So I would definitely add biding for "R" key or restart game on any movement input

**Reply by Sam-Bowden** [reply]
I'm not sure what you mean here, do you a keybinding to restart the whole game or just the current level after crashing?

**Reply by Roman-Kryvenchenko** [reply]
The level after crashing

**Reply by Sam-Bowden** [reply]
Ah OK, yeah that would be a good addition - do you want to implement this yourself as well? It might be a good place to start before implementing the settings menu to get used to Rust and the Bevy engine.

**Reply by Roman-Kryvenchenko** [reply]
Sure, then I'll create a task for it 😉
